src/tools/status_tool.cpp

```diff
@@ -162,7 +162,8 @@ void StatusTool::PrintHelp(const std::string& cmd) {
 
 int StatusTool::SpaceCmd() {
     SpaceInfo spaceInfo;
-    int res = GetSpaceInfo(&spaceInfo);
+    std::vector<LogicalpoolSpaceInfo> lpoolspaceinfo;
```

Comment:
lpoolspaceinfo can be a member of SpaceInfo.IMO:map or unordered_map is better, but there is nothing wrong with vector.